### PR TITLE
Changing where needed to reference Blue Fire instead of Fireslime

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

This PR changes the places needed to reference Blue Fire instead of Fireslime

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [ ] My PR does not decrease the code coverage, or I have __a very special case__ and explained on the PR description why this PR decreases the coverage.
- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [ ] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [ ] I read and followed the [Flame Style Guide].
- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [ ] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [ ] No, this is *not* a breaking change.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flame Style Guide]: https://github.com/flame-engine/flame/blob/main/STYLEGUIDE.md

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ However there is another
 root which showcases most of the features that exist in Flame.
 
 If you want a more full-fledged game, please check:
- - [BGUG](https://github.com/fireslime/bgug): FireSlime's open source fast paced side-scrolling
+ - [BGUG](https://github.com/fireslime/bgug): Blue Fire's open source fast paced side-scrolling
```
Comment (spydon) [inline FAQ.md:57]: Does the link here have to change too?

**Comment by spydon** [comment]
The LICENSE files are still needed right?

**Comment by wolfenrain** [comment]
> The LICENSE files are still needed right?

They are needed for publishing to pub.dev to have proper licensing resolution

**Comment by erickzanardo** [comment]
> The LICENSE files are still needed right?

Yes, they are all now links to a single file, I just made this a standard as some were files and some were links.

**Review by spydon — COMMENTED** [review]
> > The LICENSE files are still needed right?
> 
> Yes, they are all now links to a single file, I just made this a standard as some were files and some were links.

Maybe we should do the other way around since `publish` doesn't work on Linux when they are symlinks? :)

File: FAQ.md (hunk 1 above)
Comment (erickzanardo) [reply FAQ.md:57]: Yeah, good catch, missed that one

**Comment by erickzanardo** [comment]
> > > The LICENSE files are still needed right?
> > 
> > 
> > Yes, they are all now links to a single file, I just made this a standard as some were files and some were links.
> 
> Maybe we should do the other way around since `publish` doesn't work on Linux when they are symlinks? :)

It works fine on the MacOS, but I am ok in changing, I made them all links because I thought that was the standard we wanted. I will change them all to files so Linux can be happy as well

**Review by luanpotter — APPROVED** [review]
LGTM

File: i18n/README-RU.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ __ВНИМАНИЕ__: Мы работаем над доведением Flame д
 
 ## Благодарности
 
- * [Fireslime](https://fireslime.xyz), команда ответственная за поддержку Flame.
+ * [Blue Fire](https://patreon.com/bluefireoss), команда ответственная за поддержку Flame.
```
Comment (st-pasha) [inline i18n/README-RU.md:124]: Also line 79 in this file (unfortunately, can't add a comment there)

**Comment by st-pasha** [comment]
Just did a full-project search for "fireslime", and there are 17 mentions of it left.

File: i18n/README-RU.md (hunk 2 above)
Comment (erickzanardo) [reply i18n/README-RU.md:124]: Done

File: FAQ.md (hunk 1 above)
Comment (erickzanardo) [reply FAQ.md]: Fixed

**Review by spydon — APPROVED** [review]
LGTM! Was there no changes needed in the spanish readme?

**Comment by erickzanardo** [comment]
> LGTM! Was there no changes needed in the spanish readme?

I didn't realised that was merged already, thanks for pointing it out

**Pull request merged**